packer_templates/amazonlinux/amazon-2-x86_64.json

```diff
@@ -3,13 +3,13 @@
     {
       "guest_additions_path": "VBoxGuestAdditions_{{.Version}}.iso",
       "headless": "{{ user `headless` }}",
-      "http_directory": "http",
```

Comment:
I've been meaning to do this for a while. It makes a lot of commands much simpler.